src/aliases/api/types/alias.go

```diff
@@ -34,7 +34,7 @@ func FormatEntityAliases(ents []aliasent.Alias) []Alias {
 	return als
 }
 
-type AliasValue string
+type AliasValue []string
 
 type AliasKey string
```

Comment:
minor: I still personally think that this habit of redeclaring types in Go is a bit overkill as using string seems fine.

Reply:
Yeah, I think I overdid it as well, it was to validate the strings potentially, but as those data are short lived and validation is done by go-playground, having a dedicated type is overkill.

Reply:
done